daft/runners/ray_runner.py

```diff
@@ -350,8 +350,10 @@ def single_partition_pipeline(
     partial_metadatas: list[PartitionMetadata],
     *inputs: MicroPartition,
 ) -> list[list[PartitionMetadata] | MicroPartition]:
-    set_execution_config(daft_execution_config)
-    return build_partitions(instruction_stack, partial_metadatas, *inputs)
+    with with_execution_config(
```

Comment:
Nit: could we change `with_execution_config` to something like `daft_execution_context` . currently the `with with_execution_config` has a double `with with` that looks a little weird on first glance

Reply:
Yeah I agree, changed to `*_config_ctx`